tests/integration/renderer/test_due_diligence.py

```diff
@@ -99,3 +100,15 @@ def test_rddd001_initial_state(dash_duo):
 
     dash_duo.percy_snapshot(name="layout")
     assert dash_duo.get_logs() == [], "console has no errors"
+
+
+@pytest.mark.parametrize("child", [0, [0]])
+def test_rddd002_falsy_child(dash_duo, child):
+    app = dash.Dash(__name__)
+    app.layout = html.Div(id="falsy-wrapper", children=0)
```

Comment:
Shouldn't this be `children=child` ?

Reply:
thanks - combined 2 tests into one here but missed that key step! https://github.com/plotly/dash/pull/1506/commits/c24b3829e1ec068c9f5494a9ba8f19bf09ad2004